src/components/MyGames.js

```diff
@@ -1,40 +1,31 @@
 import React, { useState } from "react";
-import {
-  StyleSheet,
-  View,
-  Button,
-  Pressable,
-  Image,
-  Modal,
-} from "react-native";
 import { FlatList, TextInput } from "react-native-gesture-handler";
-import { userGames } from "../../mock-data/MockGamesList";
 import GameDetailsScreen from "./GameDetailsScreen";
+import { StyleSheet, View, Pressable, Image, Modal } from "react-native";
 
-const MyGames = ({ navigation }) => {
+const MyGames = ({ userGames, addGame, removeGame }) => {
   const [modalVisible, setModalVisible] = useState(false);
   const [selectedGame, setSelectedGame] = useState(null);
   const [searchInput, setSearchInput] = useState("");
-  const [displayedGames, setDisplayedGames] = useState(userGames);
-  // displayedGames will be set to the user's gameList, which I assume
-  // will be passed from props.
+  // const [myGames, setMyGames] = useState(userGames);
+  // myGames will be set to the user's gameList, which I assume
+  // will be passed from props or fetched with useEffect.
+
   const inputHandler = (enteredText) => {
     setSearchInput(enteredText);
     const filteredGames = userGames.filter((game) =>
       game.title.toLowerCase().includes(enteredText.toLowerCase())
     );
-    if (enteredText === "") {
-      setDisplayedGames(userGames);
-    } else {
-      setDisplayedGames(filteredGames);
-    }
+    enteredText === "" ? setUserGames(userGames) : setUserGames(filteredGames);
```

Comment:
```suggestion
    (enteredText === "") ? setUserGames(userGames) : setUserGames(filteredGames);
```
Just a readability thing :)

Reply:
Prettier removes these parens unfortunately =[